content/blog/2024-05-22-rustl-2024-unconference.md

```diff
@@ -0,0 +1,143 @@
++++
+title = "Report on the RustNL 2024 Unconference"
+authors = ["Olivier Faure"]
+date = "2024-05-23 18:00:00"
++++
+
+On May 7 and 8, 2024, folks from the Linebender team went to [the RustNL conference](https://2024.rustnl.org/) in Delft.
+We listened to talks, Raph gave one, Rik Arends did the "hot reloading my entire editor live" thing that makes every other Rust GUI developer jealous, it was an all-around good time.
+
+(Videos for individual talks aren't up yet, but there are livestream recordings of [Day 1](https://www.youtube.com/watch?v=XLefuzE-ABU&t=23721s) and [Day 2](https://www.youtube.com/watch?v=521NfGf7AR0&t=19109s) on Youtube.)
+
+After RustNL, though, another less-documented event called [The Unconference](https://2024.rustnl.org/unconf/). It was from roughly the same organizers and took place in the same city, but the format was fairly different.
+
+The Unconference wasn't livestreamed, and I haven't found any public discussion of it, so I thought I'd take the time to describe a bit of what was going.
+
+
+## The format
+
+The "Unconference" was two days of loosely-structured chats between members of various Rust project teams where people talked about ecosystem collaboration, complained to Lang Team members that their favourite feature wasn't implemented yet, [went to pester everyone else about variadic generics](https://poignardazur.github.io/2024/05/25/report-on-rustnl-variadics/), and other good fun.
+
+Attendees were split into three teams:
+
+- The Rust project.
+- GUI and Applications.
+- Embedded development.
+
+Discussions inside of these teams was mostly self-organized (more on that later), with cross-team discussion mostly taking place during lunch and recesses.
+
+
+## The GUI team
+
+As a member of the Linebender project, I was mostly exposed to the GUI team's proceedings.
+
+The main topics of interest were:
+
+- Build tooling (especially for Android and other non-desktop platforms).
+- Text layout and edition.
+- Winit adoption.
+- Accessibility.
+- Potential for further collaboration.
+- Requests for new Rust features.
+
+Note that I'm skipping a lot topics on the agenda, and other side discussions.
+It feels like those topics were the most prominent during those two days.
+
+There was also a fair amount of informal chatter around LogLog Games' incendiary [Leaving Rust gamedev after 3 years](https://loglog.games/blog/leaving-rust-gamedev/) article.
```

Comment:
```suggestion
There was also a fair amount of informal chatter around LogLog Games' [Leaving Rust gamedev after 3 years](https://loglog.games/blog/leaving-rust-gamedev/) article.
```

I think this term is too strong without a lot of reason to be.

Reply:
Incendiary is the wrong word, but that article is brazenly critical.